src/lib/utils/productionHelpers.spec.ts

```
@@ -0,0 +1,100 @@
+        "2026-08",
+      ),
+    ).toBe(50);
+  });
+});
+
+// isEffectivelySoldOut is validated here against the real on-chain numbers,
+// since it is the other half of the "dust" fix.
+describe("isEffectivelySoldOut (real on-chain supply)", () => {
+  it("treats ALB-WR1-R2's 2,000 wei of leftover supply as sold out", () => {
+    // maxSupply 36,000 - minted 35,999.999999999999998
+    expect(isEffectivelySoldOut(36000 - 35999.999999999999998)).toBe(true);
```

ESLint

[error] This number literal will lose precision at runtime.

(no-loss-of-precision)